internal/adminapi/konnect.go

```diff
@@ -28,6 +28,7 @@ type KonnectConfig struct {
 	LicenseSynchronizationEnabled bool
 	InitialLicensePollingPeriod   time.Duration
 	LicensePollingPeriod          time.Duration
+	DisableConsumersSync          bool
```

Comment:
```suggestion
	ConsumersSyncDisabled      bool
```
Maybe like that to follow naming convention of other field `LicenseSynchronizationEnabled`?